src/App.jsx

```diff
@@ -1,9 +1,29 @@
 import './App.css'
+import {Routes, Route} from 'react-router-dom'
+import ReviewList from './Components/ReviewList'
+import Nav from './Components/Nav'
+import Header from './Components/Header'
+import { useEffect, useState } from 'react'
+import { getAllCategories } from './utils/api'
 
 function App() {
 
+  const [allCategories, setAllCategories] = useState([])
+
+  useEffect(() => {
```

Comment:
Wouldn't it make more sense to have the navigation component hold this state? Why does it need to be this high?

Reply:
I'll need access to the allCategories for the filters on a later ticket. Is it okay to leave here or will I need to move up?